sdks/java/extensions/sql/zetasql/src/main/java/org/apache/beam/sdk/extensions/sql/zetasql/SupportedZetaSqlBuiltinFunctions.java

```diff
@@ -475,17 +512,22 @@
 
           // FunctionSignatureId.FN_RANGE_BUCKET, //  range_bucket(T, array<T>) -> int64
 
-          // FunctionSignatureId.FN_RAND, // rand() -> double
+          FunctionSignatureId.FN_RAND // rand() -> double
```

Comment:
This one scares me just a little. In particular, I'm worried that we might accidentally optimize such that we end up with a constant. Is that something we need to test? https://xkcd.com/221/

Reply:
Thanks for catching this! I will disable this for now, and enable it later once we have a thorough test.